src/managers/input/InputManager.cpp

```diff
@@ -295,7 +295,7 @@ void CInputManager::mouseMoveUnified(uint32_t time, bool refocus, bool mouse) {
                 if (BOX) {
                     surfacePos         = BOX->pos();
                     pFoundLayerSurface = HLSurface->getLayer();
-                    pFoundWindow       = HLSurface->getWindow();
+                    pFoundWindow       = g_pCompositor->m_pLastWindow.lock();
```

Comment:
HLSurface->getWindow() ? HLSurface->getWindow() : g_pCompositor->m_pLastWindow.lock()

you mean?

Reply:
then check if it's not hidden.